<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serverWidget</class>
 <widget class="QWidget" name="serverWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>serverWidget</string>
  </property>
  <widget class="QTextEdit" name="textEdit_readText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>501</width>
     <height>251</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_writeText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>501</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_port">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>端口：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_createRoom">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>350</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>创建聊天室</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_addUser">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>350</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>添加用户</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
